xformers/csrc/attention/hip_fmha/ck_tiled_headdim_switch.h

```diff
@@ -9,6 +9,46 @@
 #include <ck_tile/core.hpp>
 #include <stdexcept>
 
+#ifndef FMHA_SUPPORT_MAX_HEADDIM_128
```

Comment:
Thanks for adding this! Checking if we want to only have head dim = 128 support (to save compile time), not 64, 32, 256, any easy way to configure this?

Reply:
Use `unset MAX_JOBS`,  the compiling is very fast.   Even though it is easy to only build for dim == 128,  we don't like do this,  since we are not very confident with our building since the scripts provided for verifying under tests/ are not specifically prepared for dim128.  You know, for any change in the codes,  we always try to run the following scripts to verify that every-thing is correctly running:
```bash
#> pytest tests/test_mem_eff_attention.py::test_forward
#> pytest tests/test_mem_eff_attention.py::test_backward
#> pytest tests/test_mem_eff_attention.py::test_dropout
#> pytest tests/test_mem_eff_attention.py::test_dropout_backward_ck
```